### ISSUE TITLE
Game page mobile view: Review tile order

### ISSUE THREAD

**Issue by nguillaumin** [atari-legend/atari-legend#102]
As noted by @stgraveyard the tile order does not make sense on small screens, because we get the screenshot first, then all the menus, and then finally the game information and releases. It would make mores sense to have screenshot first, then the game info & releases, and then the menus.

**Comment by nguillaumin** [comment]
@stgraveyard I had a look at this. The only changes I can do is by column, i.e. I can only swap the columns order. I tried to put the first column first on mobile:

![1](https://user-images.githubusercontent.com/762512/111866246-6e9a2080-896c-11eb-98b7-7df148036a7e.png)

I'm not sure it's better? I think we would have to review which tiles go into which columns and consider the mobile experience. Maybe:

```
Game info | Screenshots | Releases
Boxscan   | Reviews     | Comments
          | Menus       | Submit info
```

This way on mobile it would give:

```
Game info
Boxscan
Screenshots
Reviews
Menus
Releases
Comments
Submit info
``` 
?

**Comment by stgraveyard** [comment]
Hi Nico,

Let's test it. Honestly, moving the boxscans to the left underneath the game info might even be better than it is now. 

Cheers,
Maarten

**Comment by nguillaumin** [comment]
I have changed it, check it on the menus test site in a few minutes...

**Comment by stgraveyard** [comment]
Hi Nico,

I like it in **wide mode**. I think this new layout makes more sense. OK, it is new so I need a moment to get used to it, but I like it. 

However, there is a bug in 2 colom mode : 

![image](https://user-images.githubusercontent.com/4339294/113358617-6a93d880-9346-11eb-848f-52a39418dc56.png)

This is for game lotus 2. Make window smaller and go to 2 column. 

Still, on a phone, I think the page is very flawed. I know this is a hard page, I spent hours on this, and I am not asking you to change it, I'm just commenting. I think in my old, crappy, non standard code, I made it so that in every view (wide, 2 column, 1 column) the order of the tiles was different. It was very difficult and not well programmed, but it worked. 
When you look at 1 column now, for starters the game title is gone, because we begin with the game info. 
Don't get me wrong, the order of the tiles makes way more sense to me than before, still I think it should start with screenshots, then info, then box ... 

I don't know what to do here. We can leave it for now, but I do think this page in particular needs some more love for mobile views ... 

Cheers,
Maarten

**Comment by nguillaumin** [comment]
Yeah I agree it does not work well. The "bug" is not a bug, it's because the next "column" (containing releases, ...) must go after the other ones, it cannot flow on the left.

The problem is that are constrained in a column layout. I can change the order of the columns (1-2-3 / 1-3-2 / 2-1-3 / ...), and I can change the order of the tiles within a column. But what is not possible without repeating the content in the HTML (which is a big no-no) is to assign a tile in a different column. I tried to summarize it here:

![layout](https://user-images.githubusercontent.com/762512/113404673-d27b0b00-93a8-11eb-94c0-87aa5f47cf1b.png)

Basically we can change the order (green / blue / yellow) and within one column (say blue) we can change the order of the tiles. But we cannot move a tile to another column. If you find a combination that would work in the 3 screen sizes let me know :wink: but I'm not sure it exists.

So I reverted it and tried something else, which is to only show the first 3 menus with a link to show the rest. This way it does not "push down" the other tiles on mobile. Have a look, let me know what you think?

**Comment by stgraveyard** [comment]
Hi Nico,

This is indeed much better, I do think game info is too low, but if this gets released I approve.

However, I have one more remark/question. I understand repeated HTML is a no - go for a hardcore coder as you ;-) BUT ... Could we maybe make a hotspot on a resolution, when we go to 1 column, add the game name to the game info box instead of the screenshot box. If we do this, and we change the order of the colors back to what we had (game info and below that the boxscans), don't we have the best solution then?

Maarten

**Comment by nguillaumin** [comment]
Thanks, it's good idea! I have implemented it, please have a look. With this new order, the list of menus is always "last" except on the very small screens. So I have removed the "View more..." except on very small screens. This ways all menus are always displayed when it's the last section of the as it does not push other content down.

**Comment by stgraveyard** [comment]
Nico, we are getting there, but with this in place, with a little shake in the order of the tiles make it even better? 

I mean this one you did on a second try (in small view) 

Game info
Boxscan
Screenshots
Reviews
Menus
Releases
Comments
Submit info

So moving the boxscan to the releases and releases to the boxscan. 

no?

**Comment by nguillaumin** [comment]
It's possible, but it will also affect the desktop view, we just have to be aware of this. I tried it, and also moved the comments on the right as per your list. Check on the menus website and let me know?

**Comment by stgraveyard** [comment]
Nico,

I personally really like this flow. In tabblet mode (2 columns) it makes the least sense, but so be it. 
Check out the game Zool for yourself as an example. In fullscreen we need to get used a bit about the replacements, but still I think this makes more sens. In 1 column it is perfect. 
So for me, this is as good as it gets. If others think differently, I will follow of course.

I do have 1 remark, which maybe can be tweaked just a bit before we go live? 
When you click on the release of a game, the tiles are still in the old order AND also in 1 column mode on this page, the game title is gone. So also here, a little tweak in that resolution is needed regarding the placement of the game title. 

Apart from this, it will be an amazing kick ass release! Thank you SO much!

**Comment by nguillaumin** [comment]
Thanks. I tried to handle the game title now, in a similar way as the game page. For the order however I would prefer to keep as-is as this is the release page, it makes sense to me to have the release description first.

**Comment by stgraveyard** [comment]
Fair enough. Sounds good to me and the game title is now in place! 

I guess we are good to go for release? ;-)
